### DISCUSSION TITLE
Nextflow nf-core scrnaseqの使い方を教えていただけないでしょうか

### DISCUSSION THREAD

**Discussion by hkinoshita54** [nig-sc/nigsc_homepage2#412] [Q&A] [answered]
nf-core scrnaseq
https://nf-co.re/scrnaseq/2.4.1
を使用したいのですが

コマンド実行後qwの状態が続き、その後ネットワークが切れてしまい、再接続して確認するとqwも消えてしまっているという状況です。

使用したコマンド

~/nextflow run nf-core/scrnaseq --input 2023_Shiokawa/data/samplesheet.csv --outdir 2023_Shiokawa/data/out --genome GRCm38 --aligner cellranger --protocol 10XV3 -profile sge -resume --max_memory 128.GB -r 2.4.1 -bg

.nextflow/assets/nf-core/scrnaseq/nextflow.config　の　profile{}　内に下記を追記しております。

sge {
        process{
            cpus = 16
            executor   = 'sge'
            penv = 'mpi'
            clusterOptions = "-cwd -S /bin/bash -l s_vmem=4G -l mem_req=4G -l epyc -o stdout -e stderr -N scrnaseq"
        }
        singularity.autoMounts = true
        docker.enabled         = false
        singularity.enabled    = true
        podman.enabled         = false
        shifter.enabled        = false
        charliecloud.enabled   = false
    }

ご存じの方がいらっしゃいましたら、Nextflowの適切な使用方法など、教えていただけないでしょうか。

**Comment by sc-fs** [comment, marked as answer]
clusterOptionsで指定するキューが「-l epyc」となっていますが、epycキューは通常のキューで、多くJOBが待ち合わせの状態です。
稼働状況は以下を参照ください。
https://sc.ddbj.nig.ac.jp/operation/

処理が１時間以内であれば、「-l short」を指定して見てください。

参考
https://webifry.hateblo.jp/entry/2021/11/17/142759

**Reply by hkinoshita54** [reply]
お返事ありがとうございます。scrnaseq内部でcellrangerを使用するので、1時間以内では終わらないと思われます。
Nextflowを直接使うのではなく、各処理を個別にJOBとしてsubmitするようにいたします。
